src/Pact/Types/ChainMeta.hs

```diff
@@ -108,7 +117,6 @@ instance HasPlafMeta () where
 
 data PublicData = PublicData
   { _pdPublicMeta :: PublicMeta
-  , _pdChainId :: Word32
```

Comment:
Would a Text-based ChainId still work with ScalableBFT?

Reply:
That's what we'd need to figure out. I'd hesitate to make the change before considering what it's going to do. It may be worth filing an issue on the private chain to migrate to a different format, but that would be a serious breaking change for our clients.

Reply:
Resolving this for now:
Per our private conversation, "ScalableBFT needs a text chain id in the future". But the issue for that needs to be fleshed out.